dash/development/base_component.py

```diff
@@ -163,6 +165,14 @@ def __init__(self, **kwargs):
 
             setattr(self, k, v)
 
+    def set_random_id(self):
+        if not hasattr(self, "id"):
+            rd = random.Random(0)
```

Comment:
I think this needs to be set in the global scope on import, otherwise this is going to produce the _same_ random ID each time it is called (because it's creating a new instance with the same seed)